terraform_files/vsphere/main.tf

```diff
@@ -0,0 +1,117 @@
+terraform {
+  required_providers {
+    vsphere = {
+      source = "hashicorp/vsphere"
+      version = "2.0.2"
+    }
+  }
+}
+
+locals {
+  hasISO = var.iso_download_path != "dummy" && var.iso_download_path != null
```

Comment:
In what case would we want a `dummy` download path?

Reply:
never it's just a marker

Reply:
please read the documentation below